Jenkinsfile.kola.gcp

```diff
@@ -96,14 +96,13 @@ podTemplate(cloud: 'openshift', label: pod_label, yaml: pod) {
                   gcp_project=\$(jq -r .project_id \${GCP_KOLA_TESTS_CONFIG})
                   # use `cosa kola` here since it knows about blacklisted tests
                   cosa kola run \
-                      -b fcos -j 5 \
```

Comment:
Is dropping the parallelism intentional?

Reply:
yeah I basically made the arguments match what we're doing for AWS, which may be wrong...

https://github.com/coreos/fedora-coreos-pipeline/blob/ad2579f2a946d582f19f4f93e8131345666e151f/Jenkinsfile.kola.aws#L88

Reply:
There's only one test right now, so there's nothing to parallelize :)